demos/multichannel_demo/yolo_v3/README.md

```diff
@@ -0,0 +1,129 @@
+# Multi-Channel Yolo v3 C++ Demo
+
+This demo provides an inference pipeline for multi-channel yolo v3. The demo uses Yolo v3 Object Detection network. You can follow [this](https://docs.openvinotoolkit.org/latest/_docs_MO_DG_prepare_model_convert_model_tf_specific_Convert_YOLO_From_Tensorflow.html) page convert the YOLO V3 and tiny YOLO V3 into IR model and execute the this demo with converted IR model.
+
+> **NOTES**:
+> This demo is default for yolo v3 model, if you want to run with tiny yolo v3 model, please patch the `anchors` vector. 
+> 
+> From :
+> ```cpp
+> std::vector<float> anchors = {10.0, 13.0, 16.0, 30.0, 33.0, 23.0, 30.0, 61.0, 62.0, 45.0, 59.0, 119.0, 116.0, 90.0,
+>                               156.0, 198.0, 373.0, 326.0};
+> ```
+> To :
+> ```cpp
+> std::vector<float> anchors = {10.0, 14.0, 23.0, 27.0, 37.0, 58.0, 81.0, 82.0, 135.0, 169.0, 344.0, 319.0};
+> ```
+    
+    
+Other demo objectives are:
+
+* Up to 16 cameras as inputs, via OpenCV*
+* Visualization of detected objects from all channels on a single screen
+
+
+## How It Works
+
+On the start-up, the application reads command line parameters and loads the specified networks. The Yolo v3 Object Detection network is required.
+
+> **NOTES**:
+> * Running the demo requires using at least one web camera attached to your machine.
+> * By default, Open Model Zoo demos expect input with BGR channels order. If you trained your model to work with RGB order, you need to manually rearrange the default channels order in the demo application or reconvert your model using the Model Optimizer tool with `--reverse_input_channels` argument specified. For more information about the argument, refer to **When to Reverse Input Channels** section of [Converting a Model Using General Conversion Parameters](https://docs.openvinotoolkit.org/latest/_docs_MO_DG_prepare_model_convert_model_Converting_Model_General.html).
+
+## Running
+
+Running the application with the `-h` option yields the following usage message:
+```sh
+cd <samples_build_folder>/intel64/Release
+./multi-channel-yolo-v3-demo -h
+
+multichannel_yolo_v3 [OPTION]
```

Comment:
actual message says: multichannel_yolo [OPTION], not multichannel_yolo_v3. Please change message (if sample will not work with other yolo versions) or change readme if it can work with other versions.